components/datatypes/consent-preferences.schema.json

```diff
@@ -187,9 +244,10 @@
         "xdm:consents": {
           "title": "Consents and Preferences",
           "description": "Specific Consent and Preference Options",
+          "type": "object",
           "allOf": [
             {
-              "$ref": "#/definitions/choices"
+              "$ref": "#/definitions/user-level-choices"
             },
             {
               "properties": {
```

Comment:
is this an empty property block?

Reply:
No, you need to expand the view (by default the diff was not showing 8 lines, which made it appear like an empty block).